init.lua

```diff
@@ -64,24 +70,26 @@ function craftguide:get_tooltip(item, recipe_type, cooktime, groups)
 			item_desc = reg_items[item].description
 		end
 	else
-		return tooltip .. "Unknown Item (" .. item .. ")]"
+		return tooltip .. S("Unknown Item (@1)", item) .. "]"
 	end
 
 	if groups then
-		local groupstr = "Any item belonging to the "
+		local groupstr = ""
 		for i = 1, #groups do
 			groupstr = groupstr .. colorize(groups[i]) ..
-				  (groups[i + 1] and " and " or "")
+				  (groups[i + 1] and (" " .. S("and") .. " ") or "")
```

Comment:
I don't think this will work for all languages, instead use e.g.:
`Any item belonging to the groups: Group 1, Group 2` with the part before the `:` being translatable